tensorflow_serving/example/inception_k8s.json

```diff
@@ -1,61 +1,36 @@
-{
-  "apiVersion": "v1",
-  "kind": "ReplicationController",
-  "metadata": {
-    "name": "inception-controller"
-  },
-  "spec": {
-    "replicas": 3,
-    "selector": {
-      "worker": "inception-pod"
-    },
-    "template": {
-      "metadata": {
-        "labels": {
-          "worker": "inception-pod"
-        }
-      },
-      "spec": {
-        "containers": [
-          {
-            "name": "inception-container",
-            "image": "gcr.io/tensorflow-serving/inception",
-            "command": [
-              "/bin/sh",
-              "-c"
-            ],
-            "args": [
-              "/serving/bazel-bin/tensorflow_serving/model_servers/tensorflow_model_server --port=9000 --model_name=inception --model_base_path=/serving/inception-export"
-            ],
-            "ports": [
-              {
-                "containerPort": 9000
-              }
-            ]
-          }
-        ],
-        "restartPolicy": "Always"
-      }
-    }
-  }
-}
-
-{
-  "kind": "Service",
-  "apiVersion": "v1",
-  "metadata": {
-    "name": "inception-service"
-  },
-  "spec": {
-    "ports": [
-      {
-        "port": 9000,
-        "targetPort": 9000
-      }
-    ],
-    "selector": {
-      "worker": "inception-pod"
-    },
-    "type": "LoadBalancer"
-  }
-}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: inception-deployment
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: inception-container
+        image: gcr.io/carbon-poet-734/inception-server:0.1.0
```

Comment:
does your image include any special binaries/setups? can we update the path to gcr.io/tensorflow-serving/inception:0.2.0 (we will update the image) so that we can maintain it?